### PR TITLE
[NETBEANS-1043]:Fixed updation of method-reference upon refactor move of method

### PR DESCRIPTION
**Pull request by arusinha**
… of method

### CODE REVIEW

File: refactoring.java/test/unit/src/org/netbeans/modules/refactoring/java/test/MoveMethodTest.java [hunk 1]
```diff
@@ -28,9 +29,12 @@
 public class MoveMethodTest extends MoveBase {
 
     public MoveMethodTest(String name) {
-        super(name);
+        super(name,"1.8");
```
Comment (vikasprabhakar) [inline refactoring.java/test/unit/src/org/netbeans/modules/refactoring/java/test/MoveMethodTest.java]: Use static String variable for "1.8"

File: refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/MoveMembersTransformer.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
 import com.sun.source.util.TreePath;
 import org.netbeans.api.java.source.support.ErrorAwareTreeScanner;
 import com.sun.source.util.Trees;
+import com.sun.tools.javac.tree.JCTree.JCMemberReference;
```
Comment (rtaneja1) [inline refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/MoveMembersTransformer.java]: Use MemberReferenceTree here?

**Comment by arusinha** [comment]
I had implemented in last commit review comments of @rtaneja1 @vikasprabhakar

**Comment by geertjanw** [comment]
Any reason not to merge this?

**Pull request merged**